Sources/ErgastAPI/ErgastAPI.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+import CloudKit
+
+public struct ErgastAPI {
+    public static func circuits(for season: Season, _ completion: @escaping (Result<YearCircuits, ErgastAPIError>) -> Void) {
```

Comment:
Because the `completion` closure occurs last you don't have to name it so the `_` isn't really necessary.